docs/partials/apiExamples.md

```diff
@@ -0,0 +1,64 @@
+<!-- AURO-GENERATED-CONTENT:START (FILE:src=./../api.md) -->
+<!-- AURO-GENERATED-CONTENT:END -->
+
+## API Examples
+
+### Property Examples
+
+#### visible
+
+State of the push notification which determines if it is `visible`.
+
+<div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/visible.html) -->
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+
+```js
+// Example function
```

Comment:
This needs to be updated to auto generate the script with `apiExamples/showToast.js` as you see on line 13 and 14. There needs to be documentation explaining that every button is using this file.

Reply:
update in this commit: https://github.com/AlaskaAirlines/auro-toast/pull/27/commits/4dd55edb65c0ac21070401e7ce8c9c5f295cad6b